config/sync/field.field.node.article.field_paragraphs.yml

```diff
@@ -48,150 +33,135 @@ settings:
   handler: 'default:paragraph'
   handler_settings:
     target_bundles:
-      text_body: text_body
-      links: links
-      medias: medias
-      accordion: accordion
-      card_grid_automatic: card_grid_automatic
-      card_grid_manual: card_grid_manual
-      content_slider: content_slider
-      files: files
-      hero: hero
-      language_selector: language_selector
-      material_grid_automatic: material_grid_automatic
-      material_grid_manual: material_grid_manual
-      banner: banner
-      nav_grid_manual: nav_grid_manual
-      breadcrumb_children: breadcrumb_children
-      nav_spots_manual: nav_spots_manual
-      recommendation: recommendation
-      user_registration_item: user_registration_item
-      content_slider_automatic: content_slider_automatic
-      user_registration_section: user_registration_section
-      video: video
-      filtered_event_list: filtered_event_list
-      manual_event_list: manual_event_list
-      simple_links: simple_links
-      material_grid_link_automatic: material_grid_link_automatic
-      user_registration_linklist: user_registration_linklist
-      webform: webform
-    negate: 0
+      campaign_rule: campaign_rule
+      event_ticket_category: event_ticket_category
+      opening_hours: opening_hours
+      go_linkbox: go_linkbox
+      go_material_slider_automatic: go_material_slider_automatic
+      go_material_slider_manual: go_material_slider_manual
+      go_video: go_video
+      go_video_bundle_automatic: go_video_bundle_automatic
+      go_link: go_link
+      go_video_bundle_manual: go_video_bundle_manual
+      go_images: go_images
+      go_text_body: go_text_body
+    negate: 1
```

Comment:
This is the magic key - the paragraphs that are chosen will be **excluded** rather than **included**.